### ISSUE TITLE
bundleUrl is readonly in RN 0.18

### ISSUE THREAD

**Issue by falsecz** [Microsoft/react-native-code-push#138]
cannot set readonly property bundleUrl
```objc
if ([CodePush isUsingTestConfiguration] || ![_bridge.bundleURL.scheme hasPrefix:@"http"]) {
            _bridge.bundleURL = [CodePush bundleURL];
}
```

due to yesterday refactoring of RCTBridge

see https://github.com/facebook/react-native/commit/ed4478a4ffb311b65e693a3f6ba8245644ee3b6b#diff-bac72910e72250822c478f2a68c2ace7L126
```objc
-@property (nonatomic, strong) NSURL *bundleURL;
+@property (nonatomic, strong, readonly) NSURL *bundleURL;
```

**Comment by lostintangent** [comment]
Ah good catch @falsecz! We haven't started looking into 0.18 yet, so we'll figure out how to support it officially and release an update in time for its non-RC release.

**Comment by skevy** [comment]
All that needs to happen is to import RCTBridge+Private.h

**Comment by skevy** [comment]
https://github.com/Microsoft/react-native-code-push/pull/139

**Comment by lostintangent** [comment]
This issue was fixed with #141. We'll release an update to NPM this weekend.
